terraform/tags.tf

```diff
@@ -0,0 +1,11 @@
+module "beacons_label" {
+  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
+  namespace   = "mca"
+  environment = terraform.workspace
+  name        = "beacons"
+
+  tags = {
+    Owner           = "BeaconRegistry@mcga.onmicrosoft.com"
+    GovtServiceName = "Beacons"
```

Comment:
@paulgutman  FYI on this here.  Please see @zackads Pull Request description.  Do you have a preference in what you wanted for the name of the tag you wanted to distinguish between services.

I suppose as long as everything uses the same tag, i.e: `GovtServiceName` then you will be able to search within the console for different services

Reply:
Ho, Happy to use the Govt prefix - it does make sense!